docs/Secure-Coding-Guide-for-Python/CWE-707/CWE-175/noncompliant01.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Non-compliant Code Example """
+import datetime, locale
+ 
+dt = datetime.datetime(2022, 3, 9, 12, 55, 35, 000000)
+ 
+def get_date(date):
+    # Return year month day tuple e.g. 2022, March, 09
+    return date.strftime("%Y"), date.strftime("%B"), date.strftime("%d")
+
+#####################
+# Trying to exploit above code example
+#####################
+
+CURRENT_LOCALE = 'en_IE.utf8'
```

Comment:
Getting the following on Flake:
expected 2 blank lines after class or function definition, found 1Flake8(E305)
(constant) CURRENT_LOCALE: Literal['en_IE.utf8']